pkg/hubble/defaults/defaults.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package defaults
+
+const (
+	// ServerPort is the default port for hubble server when a provided
+	// listen address does not include one.
+	ServerPort = 4244
```

Comment:
does it work with non-default port at all? how does it know which port to connect to if the notification doesn't include the port? 

```
{
  "name": "runtime1",
  "address": "10.0.2.15",
  "type": "PEER_ADDED"
}
```

Reply:
> does it work with non-default port at all? how does it know which port to connect to if the notification doesn't include the port?

That's a valid point. At the moment, it doesn't work with non-default ports. Not sure how we could change this because such information is not available to the node manager.

Eventually, this also means that the `--hubble-listen-addresses` doesn't make a ton of sense anymore and should probably be changed to `--enable-hubble-remote-service` or something like that.

Reply:
ok i guess we'll figure something out later.